### PR TITLE
Work on standardizing terminology

### PR DESCRIPTION
**Pull request by SabreCat**
Beginning the work of making standard, transparent use of terms for various stats.
- Health, not HP.
- Gold, not GP.
- Equipment, not items, gear, etc. (If we'd prefer "Gear" be standard, I could see it, simply due to common parlance--but I think it's the weaker term. Let me know. Definitely not "items" though, because that's ambiguous with pet-related stuff.)
- "Attack" for weapon experience gain, "Protection" for armor damage reduction.
- Expose the formulae for Strength, Defense, and overall experience boost and damage reduction.

Related PR on habitrpg-shared: https://github.com/HabitRPG/habitrpg-shared/pull/42

(I thought I had an issue ticket out there about this, but couldn't find it!)

### CODE REVIEW

**Comment by nafoster** [comment]
I definitely like all these changes. These terms are used more in various RPGs and are familiar to non-gamers.

File: views/shared/profiles/stats.jade [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
-  | : {{itemStat('armor',profile.items.armor)}}% Defense
+  | : {{itemStat('armor',profile.items.armor)}} Protection
 p
   strong {{itemText('head',profile.items.head)}}
-  | : {{itemStat('head',profile.items.head)}}% Defense
+  | : {{itemStat('head',profile.items.head)}} Protection
 p
   strong {{itemText('shield',profile.items.shield)}}
-  | : {{itemStat('shield',profile.items.shield)}}% Defense
+  | : {{itemStat('shield',profile.items.shield)}} Protection
```
Comment (lefnire) [inline views/shared/profiles/stats.jade:13]: Looks like we went back to Defense ([discussion thread here](https://trello.com/c/L1H6LiZ3/110-character-stats-attributes-the-class-system)), so let's discuss, but I think Defense will be more correct with the proposed class/attr system

**Comment by lefnire** [comment]
But... but... GP, LVL, HP, Exp... RPG terms... are we worried we're alienating non-RPG players?

File: views/shared/profiles/stats.jade (hunk 1 above)
Comment (SabreCat) [inline views/shared/profiles/stats.jade:13]: I've still got Defense (see line 31). Defense is the innate talent or learned skill of defending oneself: it's what the character has, and fits perfectly with the scheme mentioned in the discussion thread. Protection is a characteristic of armor, external to the character.

**Comment by lefnire** [comment]
fair 'nuff! ps, may not go live for a bit, I'm hitting some separate angular issues in the current code

**Comment by SabreCat** [comment]
I'm such a persuasive cuss. :tada:

**Pull request merged**